macros/stitch_user_identifiers.sql

```diff
@@ -4,21 +4,26 @@
 
 {% macro default__stitch_user_identifiers(enabled, relation=this, user_mapping_relation='snowplow_web_user_mapping') %}
     {% if enabled | as_bool() %}
-      -- Update sessions table with mapping
+
+      -- Update sessions /page_views table with mapping
       update {{ relation }} as s
-      set stitched_user_id = um.user_id
+      set stitched_user_id = um.{{ var('snowplow__user_stitching_id') }}
       from {{ ref(user_mapping_relation) }} as um
       where s.domain_userid = um.domain_userid;
+
```

Comment:
I think this change isn't actually needed - the variable gets used in the user_mapping table itself, I knew I had done it somewhere!